wordpress-mobile/WordPress-iOS
# Fix text-only empty state view layout

Fixes #19620

### Description

This PR resolves a layout issue impacting the "No media matching your search" message of the Media Picker screen.

| Before | After |
| ------- | ----- |
| <video src="https://user-images.githubusercontent.com/9609223/209958800-59906b6d-5df4-4b5f-8f46-5fa46197591a.mp4"></video> | <video src="https://user-images.githubusercontent.com/9609223/209954479-dfa08b2f-98a8-4e64-b332-529ec5577ff0.mp4"></video> |

### Test Instructions
1. Run WordPress app and log into your account
2. Tap "+" floating button then tap "Blog Post"
3. Add gallery block
4. Add media via WordPress Media Library
5. Select at least one image
6. Enter search term that will not return any results
7. Tap "Edit 1"
8. If you're not already using landscape mode, rotate now
9. Tap "<WordPress Media" to go back
10. **Expect** the "No media matching your search" message to be centered horizontally

### Regression Notes
1. Potential unintended areas of impact
The `NoResultsViewController` code was slightly modified and shouldn't cause unintended bugs. But it would be great to double check the empty state of other screen looks normal ( e.g. Notifications screen ).

<img src="https://user-images.githubusercontent.com/9609223/209965696-0a894495-2598-4201-86bb-006355488588.png" width=300 />

3. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing.

4. What automated tests I added (or what prevented me from doing so)
None.

### PR submission checklist:
